react/orders101/src/features/navbar/Navbar.jsx

```diff
@@ -0,0 +1,42 @@
+import BasketOpen from "../BasketOpen";
+import Hamburger from "../Hamburger";
+
+function Navbar() {
+  return (
+    <div class="nav container">
+      <div class="hamburger__btn-open">
+        <Hamburger />
+      </div>
+      <a class="nav__icons" href="#stores">
+        <div class="nav__icons__item container">
+          <img
+            class="searchicons"
+            src="../src/assets/geo-alt.svg"
+            alt="geo locator"
+          ></img>
+          <span>Stores</span>
+        </div>
+      </a>
+      <div class="logo">
+        <p>DIYToolWorld</p>
+      </div>
+      <a class="nav__icons" href="#ideas">
+        <div class="nav__icons__item container">
+          <img
+            class="searchicons"
+            src="../src/assets/lightbulb.svg"
+            alt="lightbulb"
+          ></img>
+          <span>Ideas & Advice</span>
+        </div>
+      </a>
+
+      <BasketOpen />
```

Comment:
We talked today about placing the "Close cart" functionality inside the `BasketClose` component instead of using `document.querySelector`. I realised that we should to handle the "Open cart" in a similar way, so figured that it would make sense to place logic in this `Navbar` component, and to move the `Cart` component in here as well. 

After all the cart gets opened and closed from the Navigation, so I would say it makes sense to keep it within this context. 

So in that case we could do something like 

```
const [isCartOpen, setIsCartOpen] = useState(false);

  const closeCart = () => {
    setIsCartOpen(false);
  }
  const openCart = () => {
    setIsCartOpen(true);
  }
```

and then pass 
```
<BasketOpen openCart={openCart} />
```

Reply:
useState for cart created and passed in BasketOpen component